src/storage_tiering.cpp

```diff
@@ -500,13 +492,15 @@ namespace irods {
         catch(const exception&) {
             const auto leaf_str = get_leaf_resources_string(_resource_name);
             metadata_results results;
-            results.push_back(
-                std::make_pair(boost::str(
-                boost::format("SELECT DATA_NAME, COLL_NAME, USER_NAME, USER_ZONE, DATA_REPL_NUM WHERE META_DATA_ATTR_NAME = '%s' AND META_DATA_ATTR_VALUE < '%s' AND META_DATA_ATTR_UNITS <> '%s' AND DATA_RESC_ID IN (%s)")
-                % config_.access_time_attribute
-                % tier_time
-                % config_.migration_scheduled_flag
-                % leaf_str), ""));
+            results.push_back(std::make_pair(
+                std::string{fmt::format(
```

Comment:
This `std::string` construction may be unnecessary...

Reply:
Agreed. `format` returns a std::string.